src/machine-learning/machine-learning.ts

```diff
@@ -244,42 +246,53 @@ export class Model {
     this.client = client;
   }
 
+  /** The ID of the model. */
   get modelId(): string {
     return extractModelId(this.model.name);
   }
 
+  /** The model's display name. This is the value you use to refer
+      to the model in the iOS and Android libraries. */
   get displayName(): string {
     return this.model.displayName!;
   }
 
+  /** The model's tags. */
```

Comment:
Optional: Add something about the tags being useful for grouping/filtering the list of models